TTS/utils/manage.py

```diff
@@ -298,22 +297,22 @@ def _set_model_item(self, model_name):
         model_item = self.set_model_url(model_item)
         return model_item, model_full_name, model, md5hash
 
-    def ask_tos(self, model_full_path):
+    @staticmethod
+    def ask_tos(model_full_path):
         """Ask the user to agree to the terms of service"""
         tos_path = os.path.join(model_full_path, "tos_agreed.txt")
-        if not os.path.exists(tos_path):
```

Comment:
🤔 This whole `if` is gone?

Reply:
The same condition is already checked in `tos_agreed()`, so no need to repeat it here (and if it were kept, a `return True` would be necessary after the `if` block actually).

Reply:
Ah, I see. (Also, it seems this wouldn't have taken the `COQUI_TOS_AGREED` envvar into account...